dali/pipeline/workspace/workspace.h

```diff
@@ -565,8 +565,7 @@ class WorkspaceBase : public ArgumentWorkspace {
    * @see operator_trace_map_t
    */
   DLL_PUBLIC void SetOperatorTrace(const std::string &trace_key, std::string trace_value) {
-    (*operator_traces_)[GetOperatorInstanceName()].insert_or_assign(
-            trace_key, std::move(trace_value));
+    GetOperatorTraces().insert_or_assign(trace_key, std::move(trace_value));
```

Comment:
Should we mention here that this function is not thread safe as mentioned `OperatorTraces::Get`?
(other functions touching operator_trace_map_t in this file as well)

Reply:
I don't think we assume that _anything_ in Workspace is thread-safe. Mentioning here that it isn't might create an illusion of thread-safety elsewhere.

Reply:
Understood.